packages/scene-composer/src/common/constants.ts

```diff
@@ -131,6 +131,7 @@ export const DRACO_PATH = 'https://www.gstatic.com/draco/versioned/decoders/1.4.
 export const DEFAULT_TAG_GLOBAL_SETTINGS: ITagSettings = {
   autoRescale: false,
   scale: 1,
+  disableOcclusion: true,
```

Comment:
when the customer has `autoRescale` or `scale` configured, this default settings will not be used, and the behavior will be disableOcclusion = false and changed existing behavior. you could revert the meaning of this new flag and make sure the behavior is not changed when the value is `false`

Reply:
Done